### PR TITLE
[JBPM-9939] Text contributor for @Lob String fields

### PR DESCRIPTION
**Pull request by elguardian**
Jira: https://issues.redhat.com/browse/JBPM-9939

### CODE REVIEW

**Comment by elguardian** [comment]
Jenkins retest this please

**Comment by elguardian** [comment]
Jenkins retest this please.

**Comment by elguardian** [comment]
Jenkins retest this please

**Comment by elguardian** [comment]
Jenkins retest this please.

File: jbpm-persistence/jbpm-persistence-jpa/src/main/java/org/jbpm/persistence/jpa/hibernate/PostgreSQLByteaTypeContributor.java [hunk 1]
```diff
@@ -39,11 +41,22 @@ public ByteaContributorType() {
...
+        }
+
+    }
+
     @Override
     public void contribute(TypeContributions typeContributions, ServiceRegistry serviceRegistry) {
         final Dialect dialect = serviceRegistry.getService(JdbcServices.class).getDialect();
         if (dialect instanceof org.hibernate.dialect.PostgreSQL81Dialect && Boolean.getBoolean("org.kie.persistence.postgresql.useBytea")) {
             typeContributions.contributeType(new ByteaContributorType());
+            typeContributions.contributeType(new TextContributorType());
```
Comment (martinweiler) [inline jbpm-persistence/jbpm-persistence-jpa/src/main/java/org/jbpm/persistence/jpa/hibernate/PostgreSQLByteaTypeContributor.java:59]: Please add a system property to enable the TextContributorType for backwards compatibility when running against an existing database. By default, it should not be enabled.

**Review by martinweiler — CHANGES_REQUESTED** [review]

**Review by afalhambra — CHANGES_REQUESTED** [review]
Great work @elguardian - just need to ask for a test :)

**Comment by elguardian** [comment]
@afalhambra merging this one and opening a new jira for that test case (discussion)

**Pull request merged**